<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>project_advanced</class>
 <widget class="QWidget" name="project_advanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>430</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Syntax</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="l0">
        <property name="text">
         <string comment="@label:textbox">Accelerator:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_Accel</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="kcfg_Accel">
        <property name="toolTip">
         <string>Usually '&amp;', but may also be '_' for GTK apps.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l1">
        <property name="text">
         <string comment="@label:textbox">Markup:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>kcfg_Markup</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="kcfg_Markup">
        <property name="toolTip">
         <string>Regular expression of markup. It is used for translation memory matching and for 'Insert tag' feature.

A default value (suitable for XML-based texts) is:
(&lt;[^&gt;]+&gt;)+|(&amp;[A-Za-z_:][A-Za-z0-9_\.:-]*;)+

It is only used for gettext PO files, as XLIFF format has its own way of specifying where markup is.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l2">
        <property name="text">
         <string comment="@label:textbox">Word wrap:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_WordWrap</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="kcfg_WordWrap">
        <property name="toolTip">
         <string>Width used for the automatic msgfmt-like word wrap. Use the value '0' to disable the word wrap. Use -1 when translating for KDE PO Summit. 1 to 3 doesn't change anything.</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>32767</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Paths</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="l3">
        <property name="text">
         <string comment="@label:textbox">Template files folder:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>potBaseDir</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KUrlRequester" name="potBaseDir" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Usually your translation project will be a subproject of a project
for translating same texts into multiple languages.

Set this to path of a folder containing empty translation files
(i.e. files without translation into any language)
shared among all subprojects.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l4">
        <property name="text">
         <string comment="@label:textbox">Branch folder:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>branchDir</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KUrlRequester" name="branchDir" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string comment="this message contains text from documentation, so use its translation when you're translating docs">This setting is for Sync Mode.

Sync Mode may be used to make changes to translation
for two branches simultaneously.

Set this to path that corresponds to root folder of the branch project,
and Secondary Sync view will automatically open files from branch.
Then, each time you make changes in files of your main branch,
they will automatically be replicated to the branch
(of course, if it contains the same English string).

See documentation for more details.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l5">
        <property name="text">
         <string comment="@label:textbox">Branch template files folder:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>branchDir</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KUrlRequester" name="potBranchDir" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string comment="this message contains text from documentation, so use its translation when you're translating docs">This setting is for Sync Mode.

Sync Mode may be used to make changes to translation
for two branches simultaneously.

Set this to path that corresponds to root folder of the branch project templates folder,
and Secondary Sync view will automatically create files from branch.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l6">
        <property name="text">
         <string comment="@label:textbox">Alternate translations folder:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>altDir</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KUrlRequester" name="altDir" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string comment="this message contains text from documentation, so use its translation when you're translating docs">Set this to path of a folder with structure similar to Root Folder.

On each translation file open, a corresponding file in Alternate translations directory will be looked up and,
if it is found, it will be used for Alternate Translations view.

You will likely want to use translations of another target language (i.e. another subproject), which is close to yours.
Also, you can use other subproject's translations either immediately, or have them pretranslated to your language by machine.

See documentation for more details.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="RelPathSaver" name="kcfg_PotBaseDir"/>
   </item>
   <item>
    <widget class="RelPathSaver" name="kcfg_BranchDir"/>
   </item>
   <item>
    <widget class="RelPathSaver" name="kcfg_PotBranchDir"/>
   </item>
   <item>
    <widget class="RelPathSaver" name="kcfg_AltDir"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RelPathSaver</class>
   <extends>QLineEdit</extends>
   <header>prefs.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
